src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -195,6 +263,22 @@ private static void StartBridge(CommandLineArguments commandLineArgs)
                 Environment.Exit(1);
             }
 
+            string resourceFolder = commandLineArgs.BridgeConfiguration.BridgeResourceFolder;
+            if (String.IsNullOrWhiteSpace(resourceFolder))
+            {
+                Console.WriteLine("Starting the Bridge requires the BridgeResourceFolder to be specified.");
+                Console.WriteLine("Use either -BridgeResourceFolder:folderName or set it as an environment variable.");
+                Environment.Exit(1);
```

Comment:
Have we defined what our exit codes are somewhere? Maybe it'd be good to start sticking them into an enum if we anticipate more than just 0 and 1